server/routes/caseNotes/caseNotesView.ts

```diff
@@ -0,0 +1,32 @@
+import { TableArgs } from '../../utils/govukFrontendTypes'
+import CaseNotesPresenter from './caseNotesPresenter'
+import ViewUtils from '../../utils/viewUtils'
+
+export default class CaseNotesView {
+  constructor(private presenter: CaseNotesPresenter) {}
+
+  private get tableArgs(): TableArgs {
+    return {
+      head: [
+        { text: 'Details', classes: 'govuk-!-width-one-quarter' },
+        { text: 'Case notes', classes: 'govuk-!-width-three-quarters' },
+      ],
+      rows: this.presenter.tableRows.map(row => {
+        return [
+          { html: `${row.sentAtDateTime}<p>${row.sentBy}` },
```

Comment:
paragraph tags should contain `class=govuk-body`

also where are the closing tags?

Reply:
closing <p> tag is optional according to html: https://html.com/tags/p/

I should make this a <br> rather than <p>

Reply:
I've changed the \<p\>s to \<br\>s